modules/installation-cis-ibm-cloud.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc
+
+[id="installation-cis-ibm-cloud_{context}"]
+= Configuring DNS resolution using Cloud Internet Services
+
+IBM Cloud Internet Services (CIS) is used by the installation program to configure cluster DNS resolution and provide name lookup for the cluster to external resources.
```

Comment:
Hey @codyhoag! I was reviewing some notes for NE, spotted some DNS limitations, and wanted to check to see if you'd like to make note of those here. 

- Currently only public DNS is supported.
- IBM Cloud does not support IPV6 so dual-stack or IPV6 environments are not possible.

And I'm not certain if it applies to this file or if we should make note of it, but: 
- Application Load Balancers (ALBs) for Virtual Private Cloud (VPC) are supported, but not classic ALB

Finally:
- `ExternalTrafficPolicy` can only be set to Cluster, not Local for IBM Cloud specifically. This might need to be called out in install, but I might also need to make not of this elsewhere.

Reply:
@bmcelvee great info! I can certainly find a home for those notes in this assembly. Thanks for sharing!

Reply:
`ExternalTrafficPolicy` is hard-coded to Cluster, in PR  https://github.com/openshift/cluster-ingress-operator/pull/516

Reply:
PS I rewrote the text in the docs slides for ALBs.